packages/integration_test/android/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 18
```

Comment:
This will need to get reflected in the changelog

Reply:
changed